index.html

```diff
@@ -80,8 +80,8 @@ <h2>Attributes</h2>
         <code>RTCLatencyCategory</code>, initialized to <code>"interactive"
         </code>.</dt>
         <dd>
-          <p>This attribute affects tradeoffs between media quality and latency.
-          It is up to the browser how best to handle this preference.</p>
+          <p>This attribute affects tradeoffs between audio or video quality
+          and latency.</p>
```

Comment:
By adding this property on the RTCRtpReceiver it is assumed that we are talking about latency on the WebRTC level, not Chrome sink/render level where it would add no benefit. Would it be necessary to add paragraph to explain how increasing latency on the WebRTC level can make audio/video quality better as you have more frames to play in case of network jitter (and more optimization like lost packets re-transmission)?